src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+
+In our example, every transaction other than the initial browser page-load transaction is the child of a span in another service, which means that every root span other than the browser transaction root has a parent span (albeit in a different service). In a fully-instrumented system (one in which every service has tracing enabled) this pattern will always hold true: _the only parentless span will be the root of the initial transaction, and of the remaining parented spans, every one will live in the same service as its parent, except for the root spans, whose parents will live in a previous service_. This is worth noting because it is the one way in which transactions are not perfect trees - their roots can (and mostly do) have parents.
+
+Now, for the sake of completeness, back to our spans:
+
+- _Backend HTML/CSS/JS Request Transactions_: 1 span each
+  - 1 root span representing the entire request (child of a browser span)
+^
+- _Backend Request with DB Call Transaction_: 2 spans
+  - 1 root span representing the entire request (child of a browser span)
+  - 1 span for querying the database (parent of the database server transaction)
+^
+- _Backend Request with API Call Transaction_: 3 spans
+  - 1 root span representing the entire request (child of a browser span)
+  - 1 span for the API request (unlike with the DB call, _not_ a parent span, since the API is external)
+  - 1 span for processing the API data
+^
+- _Database Server Request Transaction_: 3 spans
+  - 1 root span representing the entire request (child of the backend span above)
+  - 1 span for the authentication query
+  - 1 span for the query retrieving data
+
+To wrap up the example: after instrumenting all of your services, you might discover that - for some reason - it's the auth query in your database server that is making things slow, accounting for more than half of the time it takes for your entire page load process to complete. Tracing can't tell you _why_ that's happening, but at least now you know where to look!
+
+`[kmclb: working w Sami on updating the illustrations to match this example. Keeping the link below just to have the example of how to format the markdown]`
 [{% asset performance/tracing-diagram.png alt="Diagram illustrating how a trace is composed of multiple transactions." %}]({% asset performance/tracing-diagram.png @path %})
+  
+### Further Examples
+
+This section contains a few more examples of tracing, broken down into transactions and spans.
+
+**Measuring a Specific User Action**
+
+If your application involves e-commerce, you likely want to measure the time between a user clicking "Submit Order" and the order confirmation appearing, including tracking the submitting of the charge to the payment processor and the sending of an order confirmation email. That entire process is one trace, and typically you'd have transactions (_T_) and spans (_S_) for:
```

Comment:
Ah, yeah. Good catch! In an earlier incarnation, these examples looked this this:

![image](https://user-images.githubusercontent.com/14812505/81416589-c4702c00-90fe-11ea-87d9-dac8316e78e8.png)

and so I left out the root spans because the listed spans were just examples, and not meant to be an exhaustive list. But I felt like it was hard to read, so I reformatted it, but lost track of the fact that the idea of them being examples was lost in the process. Will fix. Thanks for catching this!

Reply:
Okay, how does this look?

![image](https://user-images.githubusercontent.com/14812505/81419823-bb358e00-9103-11ea-8028-343436122339.png)

Reply:
Perfect